instagram-app/src/components/posts/PostContainer.js

```diff
@@ -1,13 +1,26 @@
 import React from 'react';
 import Post from './Post';
-import './PostContainer.css';
+
+import styled from 'styled-components'
+
+
+const Post_Container=styled.div`
+ display: flex;
+    flex-direction: column;
+    max-width: 501px;
```

Comment:
Considering that the name of this styled component is very similar to the actual component using it, you might want to rename it to something else.

Also, keep in mind that the naming convention for components uses PascalCase (capitalize every word, no spaces/underscores/hyphens).